diff --git a/CMakeLists.txt b/CMakeLists.txt
index 8667819..47e2a89 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -282,7 +282,7 @@ endif ( MODULES_BUILD_STATIC )
 # We need these always, for a few of the shared modules in the muse/core build,
 #  even when building muse/core as static. Otherwise it can't find them on run. Tim.
 SET(CMAKE_BUILD_WITH_INSTALL_RPATH ON)
-SET(CMAKE_INSTALL_RPATH ${MusE_MODULES_DIR})
+SET(CMAKE_INSTALL_RPATH ${CMAKE_INSTALL_PREFIX}/${MusE_MODULES_DIR})
 
 ##
 ## Just print a notice if this is OS X
