From db237697b31efe7c51b2ae7064c3c6b072e93f6a Mon Sep 17 00:00:00 2001
From: "Jason E. Hale" <jhale@FreeBSD.org>
Date: Mon, 6 May 2024 03:43:04 -0400
Subject: [PATCH 1/2] Fix build with ninja 1.12.0, fix #1276

---
 src/muse/components/confmport.cpp | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/muse/components/confmport.cpp b/muse/components/confmport.cpp
index d01fb7dee..c4744f98b 100644
--- a/muse/components/confmport.cpp
+++ b/muse/components/confmport.cpp
@@ -44,8 +44,7 @@
 #include "arranger.h"
 #include "midiport.h"
 #include "mididev.h"
-#include "midisyncimpl.h"
-#include "midifilterimpl.h"
+#include "midisyncimpl.
 #include "ctrlcombo.h"
 #include "minstrument.h"
 #include "synth.h"

From 61312213c9b12d51ef1cbaaf3347c2f25cbfaf94 Mon Sep 17 00:00:00 2001
From: "Jason E. Hale" <jhale@FreeBSD.org>
Date: Mon, 6 May 2024 03:53:29 -0400
Subject: [PATCH 2/2] Stupid GitHub ui fix.

---
 src/muse/components/confmport.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/muse/components/confmport.cpp b/muse/components/confmport.cpp
index c4744f98b..828fe2b55 100644
--- a/muse/components/confmport.cpp
+++ b/muse/components/confmport.cpp
@@ -44,7 +44,7 @@
 #include "arranger.h"
 #include "midiport.h"
 #include "mididev.h"
-#include "midisyncimpl.
+#include "midisyncimpl.h"
 #include "ctrlcombo.h"
 #include "minstrument.h"
 #include "synth.h"
